testutils/shell.py

```diff
@@ -163,6 +192,12 @@ def pktbuf(node):
     return res
 
 
+def ifconfig(node, netif=None):
+    out = node.ifconfig_list(netif=netif)
+    res = PARSERS["ifconfig"].parse(out)
+    return res
```

Comment:
Also not covered by the tests and also not sure why this needs to be its own function.

Reply:
I use it for parsing the output of "ifconfig <if>". I need that for checking that the interface is up. Also, the same pattern is called twice in the test and once in `lorawan_netif`

Reply:
ah wait, reading twice, I think I can remove it...